ci/gitlab/.gitlab-ci.yml

```diff
@@ -250,7 +250,7 @@ github-release:
       aktualizr_src-$CI_COMMIT_TAG.tar.gz
   only:
     variables:
-      - $CI_COMMIT_REF_NAME =~ /^\d\d\d\d\.\d+(-\w+)?$/
+      - $CI_COMMIT_TAG =~ /^\d\d\d\d\.\d+(-\w+)?$/
```

Comment:
How about `/^\d\d\d\d\.\d\d?(\.\d\d?)?$/` instead? We shouldn't be doing releases with dashes and arbitrary words anyway, right?

Reply:
We sometimes like to test things, so being able to do `2019.7-test` is nice.

Reply:
Also release candidates that we don't really have bothered with in the past. But maybe we should.

Reply:
I'm in favor of doing this smaller change for now, and if we need to be even more restrictive later, it's easy enough to change.